src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleShell.cs

```diff
@@ -40,7 +40,7 @@ public static int Start(RunspaceConfiguration configuration, string bannerText,
 
         /// <summary>
         /// 
-        /// Entry point in to ConsoleShell. This method is called by the MSHManagedEntrace
+        /// Entry point in to ConsoleShell. This method is called by the MSHManagedEntrace (SP!)
```

Comment:
Naïvely, I'd guess that it should be `Entrance`, but as there's no sign of this token with or without the `n` anywhere that I could find, I can't reasonably suggest adding it...